docs/tutorial.md

```diff
@@ -14,135 +13,86 @@ By the time you read this, the latest may have changed. Be sure to check the ver
 * https://rubygems.org/gems/react_on_rails
 * https://www.npmjs.com/package/react-on-rails
 
-Trying out v4 is super easy, so long as you have the basic prerequisites. This includes the basics for Rails 4.x and node version 5+. I recommend `rvm` and `nvm` to install Ruby and Node.
+##Seting up environment
+
+Trying out **React on Rails** is super easy, so long as you have the basic prerequisites. This includes the basics for Rails 4.x and node version 5+. I recommend `rvm` and `nvm` to install Ruby and Node. Rails can be installed as ordinary gem.
 
 ```
-cd <directory where you want to create your new Rails app>
-# any name you like
-rails new test-react-on-rails-3
-cd test-react-on-rails-3
-# git-extras command to make a new git repo and commit everything
-git setup
-vim Gemfile
-```
+nvm install 5.12            # download and install latest stable Node
+nvm alias default 5.12      # make 5.12 default version
+nvm list                    # check
 
-Add this line to your Gemfile:
+rvm install 2.3.1           # download and install latest stable Ruby
+rvm list                    # check
 
-```
-gem 'react_on_rails'
+gem install rails           # download and install latest stable Rails
 ```
 
+Than we need to do normal blank Rails app as following:
```

Comment:
`Thank` => `Then`
`to do` => `to make` / `to create`
probably smth like: `to create a fresh Rails application`